src/styles/components/_post-card.scss

```diff
@@ -151,3 +152,36 @@
   font: inherit;
   margin: 0;
 }
+
+// This is a special variation of the PostCard component that's only used on
+// the homepage.
+// The card needs to have a different layout only when it's between breakpoints
+// otherwise it acts just like any other card.
+// Because it would be onerous to extend the block and all of its elements, we
+// just use a descendant selector.
+@media (min-width: #{$BREAKPOINT_VALUE_MEDIUM}) and (max-width: #{$BREAKPOINT_VALUE_LARGE}) {
+  .w-post-card--featured {
+    display: flex;
+
+    .w-post-card__cover,
+    .w-post-card__blurb {
+      flex: 1;
+    }
+
+    .w-post-card__cover,
+    .w-post-card__figure {
+      margin: 0;
+    }
+
+    .w-post-card__blurb {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      padding-left: 64px;
+    }
+
+    .w-post-card__desc::before {
+      display: none;
```

Comment:
it took me a while to work out what this was (the blue underline that indicates it's a link at small sizes). I don't really understand why we need to overwrite it again but I'm happy with it. Maybe a comment?

Reply:
good call. I'll see if I can remove it.

Reply:
turns out i can't! but added a comment